README.md

```diff
@@ -38,7 +38,9 @@ unmarkComment: false
 # Comment to post when closing a stale Issue or Pull Request. Set to `false` to disable
 closeComment: false
 # Limit to only `issues` or `pulls`
-# only: issues
+only: issues
```

Comment:
Did you mean to uncomment the `only` config? This is disabled by default.

Reply:
I did; the main reason I did this was to have clarity in the README, so that the syntax highlighter would break up the comments and the new config would be easily visible. I realize thats a bad reason, but I wasn't sure if it was commented in the first place. I can reverse it if you'd like 👍

Reply:
Since this config example is used as the default for stale, I think it should remain commented to show users how they would use it, but that by default stale acts on both issues and PRs

Reply:
Oh gotcha @hiimbex. Cool, will reverse it then.